main.tf

```diff
@@ -81,7 +81,7 @@ module "cloudtrail_baseline" {
   s3_object_level_logging_buckets   = var.cloudtrail_s3_object_level_logging_buckets
   dynamodb_event_logging_tables     = var.cloudtrail_dynamodb_event_logging_tables
   lambda_invocation_logging_lambdas = var.cloudtrail_lambda_invocation_logging_lambdas
-  is_organization_trail             = local.is_master_account
+  is_organization_trail             = var.is_organization_trail
```

Comment:
Thank you for opening the PR, and sorry for the delay in responding!

I agree that adding a way to toggle the usage of the organizational trail would be beneficial.
One suggestion is to use the organization trail for the master account by default for backward compatibility.

One possible implementation is to change `var.is_organization_trail` to `var.turn_off_organization_trail` with the default value `false`, then use `local.is_master_account && !var.turn_off_organization_trail`.

In this case, the default behavior would be to turn on the organizational trail for the master account and turn off for member accounts by default, which is backward-compatible.

How do you think?

Reply:
That makes sense to me - i'll make the change shortly.  Thanks for the response!